lib/linguist/heuristics.yml

```diff
@@ -0,0 +1,385 @@
+# A collection of simple regexp-based rules that can be applied to content
+# to disambiguate languages with the same file extension.
+#
+# There two top-level keys: disambiguations and named_patterns.
+#
+# disambiguations     - a list of disambiguation rules, one for each
+#                       extension or group of extensions.
+# extensions          - an array of file extensions that this block applies to.
+# rules               - list of rules that are applied in order to the content
+#                       of a file with matching extension. Rules are evaluated
+#                       until one of them matches. If none matches, no language
+#                       is returned.
+# language            - Language to be returned if the rule matches.
+# pattern             - Ruby-compatible regular expression that makes the rule
+#                       match. If no pattern is specified, the rule always match.
+#                       Pattern can be a string with a single regular expression
+#                       or an array of strings that will be merged in a single
+#                       regular expression (with union).
+# negative_pattern    - Same as pattern, but checks for absence of matches.
+# named_pattern       - A pattern can be reused by specifying it in the
+#                       named_patterns section and referencing it here by its
+#                       key.
+# named_patterns      - Key-value map of reusable named patterns.
+#
+# Please keep this list alphabetized.
```

Comment:
Actually, there's a test for that in `test_pedantic.rb`, [`test_heuristics_are_sorted`](https://github.com/github/linguist/blob/51feda9e1f3bd1d41f19425e663e9f6f2cb640ed/test/test_pedantic.rb#L35-L39); could you update it to check the YAML file instead? We poor humans have a hard time keeping things alphabetized (speaking from experience :p).

Reply:
Updated the test.